src/utils/hsdis/binutils/hsdis-binutils.c

```diff
@@ -219,6 +219,8 @@ static void* decode(struct hsdis_app_data* app_data, const char* options) {
 const decode_func_vtype decode_func_virtual_address = &decode_instructions_virtual;
 const decode_func_stype decode_func_address = &decode_instructions;
 
+#define remaining_buflen(buf, bufsize, p) ((bufsize) - ((p) - (buf)))
```

Comment:
This shouldn't be a macro.  Among other reasons, see the style guide.

Reply:
May I have an reference to the style guide?